designer/client/src/components/toolbarSettings/defaultToolbarsConfig.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable i18next/no-literal-string */
 import { ToolbarsSide } from "../../reducers/toolbars";
 import { WithId } from "../../types/common";
-import { ButtonsVariant } from "../toolbarComponents/ToolbarButtons";
+import { ButtonsVariant } from "../toolbarComponents/toolbarButtons/index";
```

Comment:
Something is wrong with this import `toolbarButtons/index` it should be just `toolbarButtons`

Reply:
fix everywhere